unity/Holo/Assets/Editor/ModelLoad/SingleModel.cs

```diff
@@ -84,8 +91,20 @@ protected void AddLayerComponent(GameObject go, ModelLayerInfo layerInfo)
         {
             ModelLayer layer = go.AddComponent<ModelLayer>();
             layer.Caption = layerInfo.Caption;
-            layer.Simulation = layerInfo.Simulation;
+            layer.Simulation = CheckIfSimulation(layerInfo.Simulation);
         }
 
+        private bool CheckIfSimulation(string simulationFlag)
+        {
+            string[] simulationVariants = { "true", "fibre", "flow" };
+            if (simulationVariants.Contains(simulationFlag))
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            };

```

Comment:
Again, just return the conditional expression

Reply:
Done.